From e49a7873fea78c97f2e06af3cee14193939b4939 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 12 Oct 2016 04:40:15 +0100
Subject: [PATCH] Preload libxcb now with latest Steam runtime

It explicitly checks for xcb_poll_for_reply64, which must be preloaded before
libx11 to ensure it's properly resolved.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0853d44..6d34056 100644
--- a/configure.ac
+++ b/configure.ac
@@ -60,7 +60,7 @@ AC_DEFINE_UNQUOTED(STEAM_BINARY, "$STEAMBIN",
 # The LD_PRELOAD list required to integrate gracefully with the official Steam Runtime
 AC_ARG_WITH([preload-libs], AS_HELP_STRING([--with-preload-libs],
         [the preload libs to use with Steams own runtime]), [preloadlibs="${withval}"],
-        [preloadlibs="/usr/\$LIB/libX11.so.6:/usr/\$LIB/libstdc++.so.6"])
+        [preloadlibs="/usr/\$LIB/libxcb.so.1:/usr/\$LIB/libX11.so.6:/usr/\$LIB/libstdc++.so.6"])
 
 AC_DEFINE_UNQUOTED([LSI_PRELOAD_LIBS], "$preloadlibs",
                    [Libraries to preload with Steams runtime])
-- 
2.10.1

